app/src/main/res/values-ar/strings.xml

```diff
@@ -94,14 +105,6 @@
     <!-- Text for the negative action button to dismiss the Close Tabs Banner. -->
     <string name="tab_tray_close_tabs_banner_negative_button_text">أهمِل</string>
 
-    <!-- Text for the banner message to tell users about our grid view feature. -->
-    <string name="tab_tray_grid_view_banner_message">غيّر تخطيط الألسنة المفتوحة. انتقل إلى الإعدادات واختر ”شبكة“ من ”منظور الألسنة“.</string>
-    <!-- Text for the positive action button to go to Settings for auto close tabs. -->
-    <string name="tab_tray_grid_view_banner_positive_button_text">انتقل إلى الإعدادات</string>
-
-    <!-- Text for the negative action button to dismiss the Close Tabs Banner. -->
-    <string name="tab_tray_grid_view_banner_negative_button_text">تجاهَل</string>
-
```

Comment:
These come from here https://github.com/mozilla-mobile/fenix/commit/27280e6683b28d2c7a6a6fb2c0693bd91b650060#diff-5e01f7d37a66e4ca03deefc205d8e7008661cdd0284a05aaba1858e6b7bf9103L95-L101

Reply:
Ah right, given the topic on strings from you and Sebastian yesterday, it sounds like this needs to stay until it's no longer used in release.

Reply:
Should I re-add these strings in that case (with an 'UnusedResources' suppression)?

Reply:
Yes, I think it will be the safe option for now.